### ISSUE TITLE
🐛 Bug Report: a11y - Tabbing through clickable catalog nodes doesn't work

### ISSUE THREAD

**Issue by logonoff** [backstage/backstage#30402]
### 📜 Issue Labels

- [x] Please familiarize yourself with the issue labels used in this project: [LABELS.md](https://github.com/backstage/backstage/blob/master/LABELS.md)

### 🔎 Search Terms

```plain
Keyboard, DefaultRenderNode
```

### 🗃️ Project Area

Catalog

### 🔗 External Integration

N/A

### 📝 Description & Context

The nodes with onClick passed don't have the `tabindex={0}` prop set, so those nodes cannot be focused on when using a keyboard

### 👍 Expected Behavior

I expect the clickable nodes in the catalog relation graph to be keyboard focusable

### 🥾 Reproduction steps

1. Go to https://demo.backstage.io/catalog/default/component/playback-order
2. Tab around the catalog graph
3. Notice that the nodes cannot be tabbed on

### Have you read the Code of Conduct?

- [x] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

Undecided

**Comment by MihaiTabara** [comment]
Found in today's team triaging mtg. 
@logonoff thanks for filing this! are you willing to make a contribution here and if so, how might we help?

**Comment by logonoff** [comment]
> Found in today's team triaging mtg. [@logonoff](https://github.com/logonoff) thanks for filing this! are you willing to make a contribution here and if so, how might we help?

I figured it out, thanks for the traige!
